terraform/deployments/github/repos.yml

```
@@ -369,6 +369,13 @@ repos:
       additional_contexts:
         - test
 
+  govuk_content_block_tools:
+    homepage_url: "https://components.publishing.service.gov.uk"
```

```suggestion
    homepage_url: "https://rubygems.org/gems/content_block_tools"
```
Is there a reason for the discrepancy between repo name and name of the gem?

---

Yeah, the main reason is we named the gem before we were aware of the pattern of prefixing our gems with `govuk_` - Renaming gems is non-trivial unfortunately, but changing the repo name at least gets us some of the way there.